client_code/persistence.py

```diff
@@ -49,86 +49,103 @@ def __set__(self, instance, value):
         instance._delta[self._name] = value
 
 
-@classmethod
-def _search(cls, *args, **kwargs):
-    rows = anvil.server.call(f"search_{cls.__name__.lower()}", *args, **kwargs)
-    return (cls.create(store=row) for row in rows)
-
-
-@classmethod
-def _create(cls, store=None, delta=None, *args, **kwargs):
-    instance = cls(*args, **kwargs)
-    instance._store = store or {}
-    instance._delta = delta or {}
-    return instance
+class LinkedClass:
+    "A descriptor class for adding objects based on linked tables as attributes"
 
+    def __init__(self, cls, linked_column=None, *args, **kwargs):
```

Comment:
should `linked_column` be keyword only?

Reply:
I did wonder about that but couldn't see a real benefit. Can you?

Reply:
only that if someone wants to use `*args` they first have to specify `None` or `linked_column`